app/src/main/res/values/strings.xml

```diff
@@ -3,8 +3,9 @@
 
     <string name="LandingScreen_title">Current time is: %1$s</string>
     <string name="LandingScreen_buttonContent">Next Screen</string>
-    <string name="HomeScreen_title">No Workout For Today</string>
+    <string name="HomeScreen_defaultText">No Workout For Today</string>
```

Comment:
Does the homescreen only show today's workout? Maybe `No Workout Added` makes more sense?

Reply:
How about ```Create First Workout!``` ?